### ISSUE TITLE
Show a warning for "default_locale" with no correspondent in _locales folder

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/addons-linter#989]
Steps to reproduce:
1. Submit a web-extension containing a `default_value` set in the manifest.json that has no correspondent in **_locales** folder

Expected results:
A warning is displayed at validation.

Actual results:
No validation errors. User can make it to step 3 of submission process, where **is blocked** because it has no name field and cannot go forward because that field is mandatory.

Notes/Issues:
Verified on FF49(Win 7). Issue is reproducing all around AMO servers.
Screenshots for this issue: 
![ok](https://cloud.githubusercontent.com/assets/15685960/19347853/931f2452-9153-11e6-8566-91a9dc87190c.png)
![submit](https://cloud.githubusercontent.com/assets/15685960/19347858/9b7ef82a-9153-11e6-82c0-c1ddb5e487fa.png)
![required](https://cloud.githubusercontent.com/assets/15685960/19347861/9f7ac620-9153-11e6-81d0-1ef0b3049e6c.png)

**Comment by EnTeQuAk** [comment]
Potential duplicate with https://github.com/mozilla/addons-linter/issues/701 but I'll leave this one open too to make sure to test a few addons-server related integration things as well.

**Comment by ValentinaPC** [comment]
Verified on AMO-dev FF50(Win 7).
While not having a `_locales` folder in the .xpi, and a `default_locale` set to "ar", for example, user can submit the webextension and have the possibility to enter its name during submission process.
At the end, the webextension appears to be in the `default_locale` that was initially set in the manifest.json.
Is that the intended behavior?
Attaching used webextension and screenshot from public pages:
![ar](https://cloud.githubusercontent.com/assets/15685960/21256710/9a2dfde4-c37c-11e6-81b8-e0d5d71e4616.png)
[We-Heart-It-nolocale.zip](https://github.com/mozilla/addons-linter/files/656694/We-Heart-It-nolocale.zip)

**Comment by andymckay** [comment]
I'm surprised this is actually on -dev yet, it normally takes a while for addons-linter to be merged over to addons-dev. When this and issue #989 land on addons-dev, I would retest.

Of the time of writing this comment, the last tag of addons-linter was 2 hours ago and your comment was 7 hours ago. Suggesting these bug and #989 have not landed on -dev yet.

**Comment by andymckay** [comment]
This should be on -dev now according to #1029

**Comment by ValentinaPC** [comment]
According to above results, that were different from the ones received at the time when I logged this issue, was not sure if this was fixed. 
Verified as fixed on AMO-dev FF50(Win 7).
An error is shown if no correspondent locale is available in .xpi extension archive for declared `default_locale`.
Postfix screenshot:
![2016-12-19_1050](https://cloud.githubusercontent.com/assets/15685960/21306419/90100fc8-c5d9-11e6-8422-faaa6edc73f4.png)
